pkgs/top-level/linux-kernels.nix

```diff
@@ -250,7 +250,15 @@ in {
 
     akvcam = callPackage ../os-specific/linux/akvcam { };
 
-    amdgpu-pro = callPackage ../os-specific/linux/amdgpu-pro { };
+    amdgpu-pro = callPackage ../os-specific/linux/amdgpu-pro {
+      libffi = pkgs.libffi.overrideAttrs (orig: rec {
```

Comment:
This is gross, but it depends on libffi.so.7, and we don't seem to have that anywhere else.  Maybe there's a better place for it.

Reply:
Is it possible to patchelf it to use our libffi or are the ABIs too incompatible?